common/websocket/index.ts

```diff
@@ -146,7 +151,7 @@ class WebSocketService {
                 } else {
                     log.error(`Error in websocket service.`);
                 }
-                ws.terminate();
+                ws.close(CloseCodes.SERVER_ERROR, err?.message ?? 'Internal server error while operating');
```

Comment:
what happens without explicit error code, do we get 1006?

Reply:
No, it would be a normal closure so `1000` (cf. https://developer.mozilla.org/en-US/docs/Web/API/WebSocket/close#parameters). But when terminating it, as it was made before, it would result as a `1006` in the client.